crates/bevy_ecs/src/query/iter.rs

```diff
@@ -216,12 +216,24 @@ where
 
 /// An iterator over `K`-sized combinations of query items without repetition.
 ///
+/// A combination describes the number of possible arrangements of a collection of
```

Comment:
```suggestion
/// A combination is an arrangement of a collection of
```

This is an important clarification: each combination is only one element, not the "number of combinations".